domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1891,13 +1916,15 @@ public void getConnection(Handler<AsyncResult<SqlConnection>> handler) {
       }
 
       @Override
-      public void begin(Handler<AsyncResult<Transaction>> handler) {
+      public void close(Handler<AsyncResult<Void>> handler) {
 
       }
 
+
       @Override
-      public void close() {
+      public Future<Void> close() {
         // nothing to do
```

Comment:
```suggestion
```

Reply:
Returning null as a Future most likely causes a NullPointerException. The comment "nothing to do" is misleading and should be removed.

Reply:
It wasn't my comment.